src/Symfony/Component/Cache/Traits/MemcachedTrait.php

```diff
@@ -292,7 +292,7 @@ protected function doDelete(array $ids)
      */
     protected function doClear($namespace)
     {
-        return false;
+        return $this->getClient()->flush();
```

Comment:
doing that unconditionaly is a no go onmy side as it defeats the logical pool separation. But when no namespace is used, then that works for me:
`return '' === $namespace && $this->getClient()->flush();`

Reply:
Ok, so then I will update it accordingly.